composer.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Adding a `composer.json` means this project can then be added to Packagist, which then makes it considerably easier to pull it down as a dependency on a site. It also allows us to specify dependencies, which for here, relate to PHPCS.

Reply:
Exactly.  Good job.